PKGBUILD

```diff
@@ -25,13 +24,16 @@ depends=('c-ares' 'intel-mkl' 'onednn' 'pybind11' 'openssl' 'lmdb' 'libpng' 'cur
 makedepends=('bazel' 'python-numpy' 'rocm-hip-sdk' 'miopen' 'rccl' 'git'
              'python-pip' 'python-wheel' 'python-setuptools' 'python-h5py'
              'python-keras-applications' 'python-keras-preprocessing'
-             'cython')
+             'cython' 'rocblas' 'rocrand' 'rocfft' 'hipfft' 'roctracer'
+             'hipsparse' 'hipsolver' 'rocsolver' 'rocm-hip-sdk')
```

Comment:
Are these packages not included in rocm-hip-sdk which is already listed?

Reply:
The remaining packages in my new commit are not included in rocm-hip-sdk.

Reply:
Nice!, Should rocm-hip-sdk be removed as well since it is listed earlier also can you add roctracer and hipsolver next to the other rocm deps after rccl on the same line